### DISCUSSION TITLE
Component not found error in console.

### DISCUSSION THREAD

**Discussion by tanveer2884** [livewire/livewire#8704] [1. Help]
Hi, I'm using the Livewire 3 feature wire:navigate. However, occasionally when I hit a route with wire:navigate, I encounter an error stating that the component is not found. This causes my application to stop navigating and stop working until I refresh the page.

 <li>
       <a class="dropdown-item" href="{{ route('user.favorite.music') }}" wire:navigate.hover>
             Favorite Songs
         </a>
 </li>

**Comment by EdgarsJoja** [comment]
afaik, this can happen if `wire:key` or `:key` (for components) is not set properly (mising, or not unique). Usually happens for elements that are rendered in loops.
